### PR TITLE
MAINTAINERS: add @blueskyson to collaborators

### PR DESCRIPTION
**Pull request by blueskyson**
Issue [#6330](https://github.com/tldr-pages/tldr/issues/6330).
I'm willing to become a collaborator of tldr. I have fully read community roles documentation and maintainer's guide. Thank you for appreciation!

### CODE REVIEW

**Review by CleanMachine1 — APPROVED** [review]

**Comment by CleanMachine1** [comment]
@sbrl or navarroaxel please give the role.

File: MAINTAINERS.md [hunk 1]
```diff
@@ -44,6 +44,7 @@ If you are an owner of the organization, you can see an automated list
   [03 May 2021](https://github.com/tldr-pages/tldr/issues/5873) — present
 - **Patrice Denis ([@patricedenis](https://github.com/patricedenis))**:
   [10 May 2021](https://github.com/tldr-pages/tldr/issues/5919) — present
+- **Lin Cheng Chieh ([@blueskyson](https://github.com/blueskyson))**: [12 Aug 2021](https://github.com/tldr-pages/tldr/issues/6330) — present
```
Comment (marchersimon) [inline MAINTAINERS.md:47]:
```suggestion
- **Lin Cheng Chieh ([@blueskyson](https://github.com/blueskyson))**: 
  [12 Aug 2021](https://github.com/tldr-pages/tldr/issues/6330) — present
```

**Review by marchersimon — APPROVED** [review]
🎉

**Comment by blueskyson** [comment]
Thank you all!

**Pull request merged**